androidApp/src/androidTest/java/com/mbta/tid/mbta_app/android/nearbyTransit/MapAndSheetPageTest.kt

```diff
@@ -260,7 +263,7 @@ class MapAndSheetPageTest : KoinTest {
 
         composeTestRule.waitUntilExactlyOneExists(hasContentDescription("Mapbox Attribution"))
         composeTestRule.onNodeWithContentDescription("Mapbox Attribution").assertIsDisplayed()
-        composeTestRule.waitUntilDoesNotExist(hasContentDescription("Loading..."))
+        composeTestRule.waitUntilDoesNotExistDefaultTimeout(hasContentDescription("Loading..."))
```

Comment:
The only thing that seemed to help for this test was increasing the timeout. I was playing with ways to make an a higher timeout more consistent across tests and was annoyed that you can't just override the existing timeout default for `waitUntilDoesNotExist`. Don't love this, could just pass around the constant DEFAULT_TIMEOUT in places instead.

Reply:
If we wanted to, we could replace every `waitUntilDoesNotExist(` with a `waitUntilDoesNotExistDefaultTimeout(` in a custom Spotless formatter, vaguely like how we’re handling `Map.getValue` in shared code. That might be more trouble than it’s worth, though.